sv_inline.h

```diff
@@ -852,7 +852,7 @@ PERL_STATIC_INLINE UV
 Perl_SvUV_nomg(pTHX_ SV *sv) {
     PERL_ARGS_ASSERT_SVUV_NOMG;
 
-    if (SvIOK_nog(sv))
+    if (SvUOK(sv))
```

Comment:
Prior to the commit that made this inline, it was:
`#define SvUV_nomg(sv) (SvUOK(sv) ? SvUVX(sv) : sv_2uv_flags(sv, 0))`
so that commit was erroneous, and this new one fixes it.